search-app/src/cmr/search/services/parameters/converters/shapefile.clj

```diff
@@ -99,13 +104,14 @@
   (let [crs (when (.getDefaultGeometryProperty feature)
               (-> feature .getDefaultGeometryProperty .getDescriptor .getCoordinateReferenceSystem))
         properties (.getProperties feature)
-        _ (doseq [p properties] (debug (.getName p)))
-        _ (doseq [p properties] (debug (.getValue p)))
+        _ (doseq [p properties] (debug (.getName p)) (debug (.getValue p)))
```

Comment:
I left it in because it is useful to be able to turn it on if we need to debug (like when we add new formats or are trying to debug a problem shapefile).